### ISSUE TITLE
[Feature request] Minigame Rewards

### ISSUE THREAD

**Issue by SuperSmashBrony** [leoetlino/project-restoration#68]
In MM3D, some minigames give fishing hole passes as rewards in lieu of the rupees given in the original. For those who don’t play the fishing holes, like myself, these passes are useless. Would it be possible to receive the rupee rewards in addition to the fishing hole passes? If multiple rewards aren’t possible, then I’d rather just have the rupees, especially since you normally need to spend rupees to get into the fishing hole anyway.

**Comment by GeneraLight** [comment]
I agree. Fishing Passes should either be a secondary/consolation prize for mini-games, or omitted altogether if that's not possible. Winning them in unrelated minigames seems out of place and screams "try out the new fishing holes we developed!".

They should still be prizes at both Fishing Holes though and buyable at certain places like the West Cock Town shop(which looks like a fishing shop).
